docs/documentation/upgrading/topics/changes/changes-26_2_2.adoc

```diff
@@ -0,0 +1,7 @@
+== Notable changes
+
+Notable changes where an internal behavior changed to prevent common misconfigurations, fix bugs or simplify running {project_name}.
+
+=== local admin deprecation
+
+`UrlType.LOCAL_ADMIN` and the corresponding welcome theme variable `localAdminUrl` have been deprecated for eventual removal. The default welcome resource will now simply mention localhost rather than providing a URL when an admin user has yet to be created.
```

Comment:
Despite the deprecation not being a breaking change, I believe we can't do it in a patch release. I'd suggest handling the deprecation in a separate PR for 26.3.

Reply:
> Despite the deprecation not being a breaking change, I believe we can't do it in a patch release.

It seems like we can't avoid having some kind of notice in 26.2.2. If you are worried about the case of having a custom hostname provider customize LOCAL_ADMIN, they would need to know that it is no longer used by the default welcome resource.

Reply:
My belief is that it would still be a good idea to let users know that LOCAL_ADMIN is not used by default anymore, and that if they are providing their own LOCAL_ADMIN that it should be hardened. If this can't be done via a deprecation because it's a patch release, then I'd still want some change note.

cc @mabartos @abstractj @stianst